src/0.9.0/sdks-api.md

```diff
@@ -185,6 +186,69 @@ If you're using another development language, please read up on our [immugw](htt
 
 ::::
 
+# Verify state signature
+
+If `immudb` is launched with a private signing key is possible to verify with the public key each signed request.
```

Comment:
I would rephrase this to: "If `immudb` is launched with a private signing key, each signed request can be verified with the public key."